linux/LinuxProcessList.c

```diff
@@ -1504,6 +1550,14 @@ static bool LinuxProcessList_recurseProcTree(LinuxProcessList* this, openat_arg_
 
       LinuxProcessList_recurseProcTree(this, procFd, "task", proc, period);
 
+      if (ss->flags & PROCESS_FLAG_LINUX_CGROUP || hideRunningInContainer) {
+         LinuxProcessList_readCGroupFile(lp, procFd);
+         if (hideRunningInContainer && lp->cgroup && isContainerOrVMSlice(lp->cgroup)) {
+            if (!LinuxProcessList_checkPidNamespace(proc, procFd))
+               goto errorReadingProcess;
+         }
```

Comment:
```suggestion
         if (hideRunningInContainer && lp->cgroup && isContainerOrVMSlice(lp->cgroup)) {
            if (!LinuxProcessList_checkPidNamespace(proc, procFd)) {
               goto errorReadingProcess;
            }
         }
```

No need to save on those braces. While technically the style guide allows to drop them, it's usually better to favor consistency and either keep them for all of the one-line conditions or drop them for all. In this particular block I think keeping them all looks nicer overall.